client/lib/gsuite/new-users.ts

```diff
@@ -176,9 +177,11 @@ const validateNewUserMailboxIsUnique = (
  * Adds a duplicate error to each mailBox with a duplicate mailbox
  */
 const validateNewUsersAreUnique = ( users: GSuiteNewUser[] ): GSuiteNewUser[] => {
-	const mailboxesByCount: { [ mailbox: string ]: number } = countBy(
-		users.map( ( { mailBox: { value: mailBox } } ) => mailBox )
-	);
+	const mailboxesByCount = users.reduce( ( workingCount: Record< string, number >, user ) => {
+		const mailboxName = user.mailBox.value;
```

Comment:
Should we uppercase the `B` for consistency?

```suggestion
	const mailBoxesByCount = users.reduce( ( workingCount: Record< string, number >, user ) => {
		const mailBoxName = user.mailBox.value;
```

Reply:
Done! 16db851